pkg/arvo/lib/crunch.hoon

```diff
@@ -0,0 +1,361 @@
+/-  c=crunch, gs=graph-store, ms=metadata-store, p=post, r=resource
+::
+=<
+  |_  [our=ship now=@da]
+  ++  walk-graph-associations
+    |=  [=associations:ms content=? from=@da to=@da]
+    ^-  wain
+    :: graph resources in `our`; used to avoid scrying, e.g.,
+    ::  a graph `our` has left and can no longer access
+    ::
+    =/  accessible-graphs=(set resource:r)  (scry-graph-resources)
+    %-  ~(rep by associations)
+    |=  [[=md-resource:ms =association:ms] out=wain]
+    ^-  wain
+    ?>  ?=(%graph app-name.md-resource)
+    ?>  ?=(%graph -.config.metadatum.association)
```

Comment:
Now that I think about it, no-op is better, as you suggest. Friendlier. I could imagine a user passing in a mix of associations (e.g. some group and some graph), and it'd be nice to process the graphs rather than crashing. I've addressed this in f7330e5.